### PR TITLE
Add Integration ID to Metric Alert

### PR DESCRIPTION
**Pull request by taj-p**
# Intent

Piggyback off [GirtsZemitis](https://github.com/GirtsZemitis)'s great starter (https://github.com/jianyuan/terraform-provider-sentry/pull/196) to add an integration ID to metric alerts. 

We need to add an `integration_id` field to the metric alert action in order to specify the Slack Integration (in order to send alerts to Slack).

This is seen in the below example:

```tf
data "sentry_organization_integration" "slack" {
  organization = "organization"
  provider_key = "slack"
  name         = "Organization" // corresponds to Slack's name of your organisation
}

resource "sentry_metric_alert" "main" {
  // ...
  trigger {
    action {
      type              = "slack"
      target_type       = "specific"
      target_identifier = "#slack-channel"
      integration_id    = data.sentry_organization_integration.slack.id // <---- New addition
    }
    // ...
  }

```
## Prereqs

Requires https://github.com/jianyuan/go-sentry/pull/66

### CODE REVIEW

File: sentry/resource_sentry_metric_alert_test.go [hunk 1]
```diff
@@ -132,6 +132,7 @@ resource "sentry_metric_alert" "test" {
 			type              = "email"
 			target_type       = "team"
 			target_identifier = sentry_team.test.internal_id
+            integration_id    = 32
```
Comment (taj-p) [inline sentry/resource_sentry_metric_alert_test.go:135]:
```suggestion
			target_identifier = sentry_team.test.internal_id
                        integration_id    = 32
```

File: docs/index.md [hunk 2]
```diff
@@ -274,6 +274,12 @@ output "issue_alert_url" {
 # Metric alert
 #
 
+data "sentry_organization_integration" "slack" {
+  organization = "organization"
+  provider_key = "slack"
+  name         = "organization-slack"
```
Comment (taj-p) [inline docs/index.md:280]:
```suggestion
  name         = "Organization" // corresponds to Slack's name of your organisation
```

File: docs/resources/metric_alert.md [hunk 3]
```diff
@@ -35,6 +41,18 @@ resource "sentry_metric_alert" "main" {
     threshold_type  = 0
   }
 
+  trigger {
+    action {
+      type              = "slack"
+      target_type       = "specific"
+      target_identifier = "#slack-channel"
+      integration_id    = data.sentry_organization_integration.slack.internal_id
```
Comment (taj-p) [inline docs/resources/metric_alert.md:49]:
```suggestion
      integration_id    = data.sentry_organization_integration.slack.id
```

File: sentry/resource_sentry_metric_alert.go [hunk 4]
```diff
@@ -324,6 +328,9 @@ func expandMetricAlertTriggerActions(actionList []interface{}) []*sentry.MetricA
 				action.ID = sentry.String(v)
 			}
 		}
+		if v, _ := actionMap["integration_id"].(int); v != 0 {
```
Comment (taj-p) [inline sentry/resource_sentry_metric_alert.go:331]:
```suggestion
		if v, ok := actionMap["integration_id"].(int); ok && v != 0 {
```

File: sentry/resource_sentry_metric_alert_test.go (hunk 1 above)
Comment (taj-p) [inline sentry/resource_sentry_metric_alert_test.go:135]:
```suggestion
			target_identifier = sentry_team.test.internal_id
			integration_id    = 32
```

**Comment by jianyuan** [comment]
Looks good to me. Thank you, @taj-p and @GirtsZemitis!

**Pull request merged**